lib/integrations/awsoidc/list_ec2ice.go

```diff
@@ -126,11 +141,17 @@ func ListEC2ICE(ctx context.Context, clt ListEC2ICEClient, req ListEC2ICERequest
 		name := aws.ToString(ice.InstanceConnectEndpointId)
 		subnetID := aws.ToString(ice.SubnetId)
 		state := ice.State
+		stateMessage := aws.ToString(ice.StateMessage)
+		dashboardLink := fmt.Sprintf("https://%s.console.aws.amazon.com/vpc/home?#InstanceConnectEndpointDetails:instanceConnectEndpointId=%s",
+			req.Region, aws.ToString(ice.InstanceConnectEndpointId),
```

Comment:
Are you 100% sure that these endpoint IDs are URL-safe? I'd say we should URL-encode them just in case.

Reply:
Well, they should be URL safe (the same way that regions usually are).
I added a `url.QueryEncode` just to play it safe :+1: